src/Tests/CESMII.NodeSetUtilities.Tests/Properties/launchSettings.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Please remove this file from the PR. It's usually not a good idea to check in these files as we all tweak them to our liking.